src/Features/Core/Portable/AddImport/AbstractAddImportFeatureService.cs

```diff
@@ -67,7 +67,7 @@ protected AbstractAddImportFeatureService()
 
             using (session)
             {
-                if (session == null)
+                if (session == null || !RemoteSupportedLanguages.IsSupported(document.Project.Language))
```

Comment:
How would this happen?  AbstractAddImportFeatureService is only derived with C# and VB.